pkg/arch/arch.go

```diff
@@ -81,3 +81,14 @@ func HasSyscallPrefix(symbol string) bool {
 	}
 	return false
 }
+
+// Return the syscall name without arch prefix
+func TrimSyscallPrefix(symbol string) string {
```

Comment:
we seem to have CutSyscallPrefix doing the same thing

Reply:
If i'm correct `CutSyscallPrefix` in this case will turn `__x64_sys_something`  to `sys_something` where `TrimSyscallPrefix` will do return `something`